packages/babel-plugin/__tests__/stylex-transform-stylex-attrs-test.js

```diff
@@ -959,10 +946,12 @@ describe('@stylexjs/babel-plugin', () => {
           _inject2(".color-xju2f9n{color:blue}", 3000);
           ({
             0: {
-              class: "FooBar__styles.default color-x1e2nbdu"
+              class: "color-x1e2nbdu",
+              "data-style-src": "js/FooBar.react.js:3"
             },
             1: {
-              class: "FooBar__styles.default FooBar__styles.active color-xju2f9n"
+              class: "color-xju2f9n",
+              "data-style-src": "js/FooBar.react.js:3; js/FooBar.react.js:3"
```

Comment:
If you have lots of styles within the same `create` call, and then use them all on the same element (e.g., different conditionals), it produces duplicates at the moment.

Reply:
Maybe we'd want the line of the namespace key rather than the `create` call? Some `creates` are pretty large with many keys. If not, add some dev overhead to the runtime to dedupe?

Reply:
Latest iteration displays the line number of the individual style objects